e2e/testdata/otel-collector/otel-cm-full-gcm.yaml

```diff
@@ -17,9 +17,10 @@ data:
   otel-collector-config.yaml: |-
     receivers:
       opencensus:
+        endpoint: 0.0.0.0:55678
```

Comment:
Do we know why upstream switch from 0.0.0.0 to localhost? Why doesn't that work for us? Was it a security fix?

Reply:
Yes it's for CVE https://github.com/open-telemetry/opentelemetry-collector/issues/6151